modules/cli/src/main/scala/scala/cli/launcher/LauncherCLI.scala

```diff
@@ -0,0 +1,59 @@
+package scala.cli.launcher
+import dependency._
+
+import scala.build.internal.{OsLibc, Runner}
+import scala.build.options.{BuildOptions, JavaOptions}
+import scala.build.{Artifacts, Positioned}
+import scala.cli.commands.LoggingOptions
+
+object LauncherCLI {
```

Comment:
NIT NIT NIT Could we rename this to `LauncherCli`? That's the case convention we've been following in the Scala CLI code base, when "cli" is in a class name.